### PR TITLE
Call for organizers annoucement

### PR DESCRIPTION
**Pull request by xabomon**
This is a minimal page to announce our call for organizers.

Feel free to modify the document (announcement.md), or comment in this PR.

### CODE REVIEW

**Comment by xabomon** [comment]
This is part of #49

File: announcement.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 After a successful [inaugural session](inauguralsession.md) we aim to bring VVirtual Science Forum closer to its conceived goal: facilitate scientific communities organizing virtual events.
```
Comment (aggrushin) [inline announcement.md]: After a successful [inaugural session](inauguralsession.md) we aim to bring the Virtual Science Forum closer to its conceived goal: aid researchers to organizing virtual events.

File: announcement.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
 
-In order to get a step closer  we are seeking for potential session organizers.
+In order to get a step closer to it we are seeking for potential session organizers.
 The necessary information to organize a session can be found [here](organizierguide.md).
 
-We will be grateful if this annoucement will be send around to colleagues that might be interested in organizing a virtual session.
+We will be grateful if this announcement will circulate around to colleagues that might be interested in organizing a virtual session.
```
Comment (aggrushin) [inline announcement.md]: We would be grateful if you could circularte this announcement among colleagues that could be interested in organizing a virtual session.

File: announcement.md (hunk 1 above)
Comment (xabomon) [reply announcement.md]: Change has been addressed in latest commit

File: announcement.md (hunk 2 above)
Comment (xabomon) [reply announcement.md]: Address in latest commit

**Comment by xabomon** [comment]
Solved by #54 closing PR.

**Comment by xabomon** [comment]
Merge PR after edits in #54

**Pull request merged**